SU-SWS/stanford_bean_types
# Hsdo 812 Hsdo 554 hero bean

# READY FOR REVIEW

# Summary
- New Hero Bean type that provides a curtain reveal and a scrolling button option.

# Needed By (Date)
- 6/7

# Steps to Test
1. Make sure you have latest stanford_paragraph_types.
2. Checkout this branch, enable stanford_bean_types_hero, ensure its reverted.
3. Create a new bean of type 'stanford_hero_block' with a large image and/or youtube video
4. optionally add some text in the text fields.
5. save and the bean image/video contents should be visible at the very top of the page.

# What to expect
Should be similar to below
- [Curtain reveal](https://www.thecssninja.com/demo/css_reveal/)
- [Scroll Button](http://bellarmine.lmu.edu/)
- Static should just be at the top with no extras.

# Behat tests
https://github.com/SU-SWS/linky_clicky/pull/161
Before merging, we should remove the `CLICKY_BRANCH` from .travis.yml file
